Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -368,6 +368,18 @@ public bool OnlyUseDefaultSpriteName
             return !string.IsNullOrEmpty(getSpriteNameAction);
         }
     }
+
+    /// <summary>
+    /// Whether the furniture has power or not.
+    /// </summary>
+    /// <returns>True if the furniture has power.</returns>
+    public bool HasPower
+    {
+        get
+        {
+            return PowerConnection == null || World.Current.PowerNetwork.HasPower(PowerConnection);
```

Comment:
in that case HasPower should be false when not on the grid, but then alter the check for operation ` IsOperating = HasPower && canFunction;`
also the current system shows non connected unpowered items as green on line 1075